chrome-extension/content.js

```diff
@@ -93,6 +94,15 @@ function createSidebar(content) {
   commentTextarea.setAttribute('placeholder', 'Note to self');
   form.appendChild(commentTextarea);
```

Comment:
I think it would be good idea to add a label on top like you do for highlighted text. Just some text that says "Note" or something.

Reply:
okie!